### DISCUSSION TITLE
Change of sync interval reverts back to default (daily).

### DISCUSSION THREAD

**Discussion by sniff-numeral** [bitfireAT/icsx5#436] [General]
Hello,
As the title says. When I change the sync interval to something other than the default, the button reverts back to daily when I close and re-open the app. However, while the button is on daily, the app syncs at the previously selected interval (I only tested 15 minutes though). So the problem seems to be cosmetic.

**Comment by sunkup** [comment]
Thanks, we will look into it.

Tracked in #437

**Reply by sniff-numeral** [reply]
Just tried it in version 2.2.5 and it now works as it should. Thank you!

**Comment by aisknab** [comment]
This happens to me on GrapheneOS. 

Manual sync works but not automatic. No matter what interval I change it to. I have the latest 2.2.6 version.
